src/tests/ftest/rebuild/cascading_failures.py

```diff
@@ -14,7 +14,7 @@ class CascadingFailures(RebuildTestBase):
     :avocado: recursive
     """
 
-    CANCEL_FOR_TICKET = [["DAOS-2799", "targets", 8]]
+    CANCEL_FOR_TICKET = [["DAOS-6728"]]
```

Comment:
Don't think this is the correct way of using `CANCEL_FOR_TICKET` option. The ticket id should be accompanied with some condition from yaml file or the test case.